unix/w0vncserver/Pipewire.cxx

```diff
@@ -0,0 +1,411 @@
+#include "Pipewire.h"
```

Comment:
All `.cxx` files should make sure that `config.h` is included, or things misbehave on certain platforms.